packages/scene-composer/src/components/panels/ColorPicker/ColorPicker.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react';
+import { debounce } from 'lodash';
+import { SpaceBetween } from '@awsui/components-react';
+import { useIntl } from 'react-intl';
+import { ColorRepresentation } from 'three';
+
+import {
+  ColorPickerWrapper,
+  FlexibleDiv,
+  validateHex,
+  validateRgbValue,
+  hexString,
+  isValidColor,
+  DEFAULT_COLOR,
+  updateColorWithRgb,
+  getRgb,
+  ReactColorfulPicker,
+  HexInputField,
+  RgbInputField,
+} from './ColorPickerHelpers';
+
+interface ColorPickerProps {
+  color: ColorRepresentation | undefined;
+  onChange: (newColor: string) => void;
+}
+
+export const ColorPicker: React.FC<ColorPickerProps> = ({ color, onChange }: ColorPickerProps) => {
+  const { formatMessage } = useIntl();
+  const [internalColor, setInternalColor] = useState<ColorRepresentation>(color ? color : DEFAULT_COLOR);
+  const [hexInputValue, setHexInputValue] = useState<string>(color ? hexString(color) : hexString(DEFAULT_COLOR));
+
+  const handleColorfulChange = (newColor: ColorRepresentation) => {
+    setInternalColor(newColor);
+    setHexInputValue(hexString(newColor));
+    onChange(hexString(newColor));
+  };
+
+  const handleHexChange = (newColor: string) => {
+    setHexInputValue(validateHex(newColor));
+    if (isValidColor(newColor)) {
+      setInternalColor(newColor);
+      onChange(newColor);
+    }
+  };
+
+  const handleRGBChange = (newValue) => {
+    const updatedColor = updateColorWithRgb(internalColor, newValue);
+    setInternalColor(updatedColor);
+    setHexInputValue(updatedColor);
+    onChange(updatedColor);
+  };
+
+  return (
+    <ColorPickerWrapper>
+      <form onSubmit={(e) => e.preventDefault()}>
+        <ReactColorfulPicker color={hexString(internalColor)} onChange={debounce(handleColorfulChange, 100)} />
+        <FlexibleDiv>
+          <HexInputField
+            color={`#${hexInputValue.replace('#', '')}`}
+            onChange={(e) => handleHexChange(e.detail.value)}
+            label={formatMessage({ defaultMessage: 'Hex', description: 'Abbreviation for hexidecimal' })}
+            errorText={
+              !isValidColor(hexInputValue) &&
+              formatMessage({ defaultMessage: 'Invalid Hex', description: 'Form field error message' })
```

Comment:
Please implement `react-intl` for strings that appear in the UI. 

[Example here. ](https://github.com/awslabs/iot-app-kit/blob/8eee582e039a5cb2a3e25c81f822a4ecca95672c/packages/scene-composer/src/components/panels/scene-components/AnchorComponentEditor.tsx#L197)